202009.0/5bb12695-41cf-488f-971b-c9304118d522.md

```diff
@@ -11,10 +11,11 @@ Providing users with a proper on-site search user experience is often one of the
 * the right search results show up as suggestions when text is entered into the search box (completion)
 * an alternative search result is shown when a term is misspelled (spell checking)
 
+
 Furthermore, we will introduce a technique for sorting search results which ranks products higher that:
 
 * are most relevant for the search
-* exhibit better past performance (revenue, clicks, click trough rate, etc.)
+* exhibit better past performance (revenue, clicks, click trough rate, product popularity, etc.)
```

Comment:
please revert it, because it's not Spryker doc https://project-a.github.io/on-site-search-design-patterns-for-e-commerce/

Reply:
DONE